--- a/.pio/libdeps/dongle/LovyanGFX/src/lgfx/v1/platforms/esp32s3/Bus_RGB.cpp
+++ b/.pio/libdeps/dongle/LovyanGFX/src/lgfx/v1/platforms/esp32s3/Bus_RGB.cpp
@@ -95,7 +95,7 @@ namespace lgfx
     void _gpio_pin_sig(uint32_t pin, uint32_t sig)
     {
       if (pin < GPIO_NUM_MAX) {
-        gpio_hal_iomux_func_sel(GPIO_PIN_MUX_REG[pin], PIN_FUNC_GPIO);
+        gpio_iomux_out(pin, PIN_FUNC_GPIO, false);
       }
     }
